koans/1.intro.js

```diff
@@ -7,7 +7,7 @@ import {
 } from 'tape'
 
 import {
-  arrow,
+  arr,
```

Comment:
Good point. I guess I'm mostly thinking in terms of `lists` rather than `arrays`, and as I was taking a second look at the [Control.Arrow code](https://hackage.haskell.org/package/base-4.9.1.0/docs/src/Control.Arrow.html#Arrow) I realized the _lifter_ is called `arr` (which would make sense since it's short enough to be thrown in very quickly anywhere).

Maybe we should revisit the nomenclature here: should all constructors/lifters be ClassCased? I'm thinking we have `Left(a)` and `Right(b)`, should we also have `Cell(fn, child)` and `Arrow(fn)` ?

Or should we settle for lowercased shortnames such as `cell(fn, child)` or `arr(fn)` and `left(a)`, `right(b)`? I feel the latter has more of a minimalist feel that appeals to me.

Either way this PR seems a good place as any to decide this since koans would be more representative of real code than the internal usage or the tests, but we can also take a stab at the `tessellation/zazenify` branch and see how things would look with alternative nomenclatures there.